wpimath/src/main/java/edu/wpi/first/math/kinematics/HDriveWheelSpeeds.java

```diff
@@ -0,0 +1,16 @@
+package edu.wpi.first.math.kinematics;
+
+public class HDriveWheelSpeeds {
```

Comment:
Add an `HDriveWheelSpeeds::desaturate` and `toString` option following other wheel speeds classes